lib/PuppeteerSharp/Frame.cs

```diff
@@ -315,6 +316,22 @@ internal Task<JSHandle> WaitForFunctionAsync(string script, WaitForFunctionOptio
             return handle.AsElement();
         }
 
+        internal async Task<string[]> SelectAsync(string selector, object[] args)
```

Comment:
in puppeteer the `args` is called `values` and if of type `string[]` not `object[]`

Reply:
I didn't know that `params string[]` was valid, that's cool.

Reply:
params works with `T[]` 😄